.github/workflows/bcny-firebase.yml

```diff
@@ -98,3 +98,142 @@ jobs:
         with:
           name: firebase-windows-${{ matrix.arch }}
           path: ${{ github.workspace }}/BuildRoot/Library/firebase
+
+      - name: Package firebase-cpp-sdk
+        run: |
+          @"
+          <?xml version="1.0" encoding="utf-8"?>
+          <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+            <metadata>
+              <id>com.google.firebase.windows.${{ matrix.arch }}</id>
+              <version>0.0.0.0</version>
+              <title>Firebase C++ SDK</title>
+              <description>C++ Firebase SDK</description>
+              <authors>Google, Inc.</authors>
+              <projectUrl>https://firebase.google.com</projectUrl>
+              <repository type="git" url="https://github.com/google/firebase-cpp-sdk" branch="main" />
+            </metadata>
+            <files>
```

Comment:
Is it possible to put pdbs in here as well? I think these should be getting generated but don't seem to be copied anywhere.

Reply:
Sounds like a plan!